AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -20,6 +20,7 @@ turkishairlines.com,atmos-tokyo.com,radarbox.com,ciceksepeti.com,rabota.by,temp-
 ! https://github.com/AdguardTeam/AdguardFilters/issues/161301
 temu.com#?#div[role="main"] div[style] > div[style]:has(> div[style] div[js-selector="mobile-download-ui"])
 !
+flowgpt.com##div[class="chakra-portal"]
```

Comment:
This rule causes that log in button doesn't work.
<details><summary>Screenshot</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/29142494/55398efe-04f6-4839-bc47-4a424351aeff)

</details>

Maybe something like this:
```adblock
flowgpt.com#%#//scriptlet('set-local-storage-item', 'appDownloadModalDismissed', 'true')
```
can be used instead.

Reply:
Indeed, I did not check the desktop version, thank you!